compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/ComposeScene.skiko.kt

```diff
@@ -583,13 +583,24 @@ class ComposeScene internal constructor(
     }
 
     private fun processPress(event: PointerInputEvent) {
-        val owner = hoveredOwner(event)
-        if (focusedOwner.isAbove(owner)) {
-            focusedOwner?.onDismissRequest?.invoke()
-        } else {
-            owner?.processPointerInput(event)
-            pressOwner = owner
+        val hoveredOwner = hoveredOwner(event)
```

Comment:
Maybe call this `pressedOwner`, since we're reacting to a press.

Reply:
there is `pressOwner` already (not exactly match to call it name conflict, but anyway), it assigned only if all check in this function are passed

Reply:
I think that's one more reason to call the local variant `pressedOwner` or `pressOwner` - it refers to the same thing, but before we "remember" it.

Reply:
`pressedOwner` is better (for the instance variable too) because `pressOwner` can be confused to mean "the owner of a press".